pkg/kubectl/cmd/logs.go

```diff
@@ -293,13 +289,32 @@ func (o LogsOptions) RunLogs() error {
 		return err
 	}
 
+	doneChan := make(chan bool)
+	piper, pipew := io.Pipe()
 	for _, request := range requests {
-		if err := o.ConsumeRequestFn(request, o.Out); err != nil {
-			return err
-		}
+		go func(request *rest.Request) {
+			if err := o.ConsumeRequestFn(request, pipew); err != nil {
```

Comment:
does this interleave log lines between sources? does it interleave sub-line?

Reply:
is it unexpected that there is no ordering between logs from different containers (they just stream together in whatever order the lines are fetched)?